undocumented

```diff
@@ -13,139 +14,135 @@ app/store/notifications.js:145 "evict"
+app/views/topology/relation.js:406 "drawRelation"
+app/views/topology/relation.js:335 "drawRelationGroup"
+app/views/topology/relation.js:566 "addRelationDragStart"
+app/views/topology/relation.js:550 "snapOutOfService"
+app/views/topology/relation.js:429 "updateSubordinateRelationsCount"
+app/views/topology/relation.js:907 "ambiguousAddRelationCheck"
+app/views/topology/relation.js:269 "updateLinks"
+app/views/topology/relation.js:521 "snapToService"
+app/views/topology/relation.js:185 "initializer"
 app/views/topology/panzoom.js:91 "renderSlider"
 app/views/topology/panzoom.js:77 "componentBound"
 app/views/topology/panzoom.js:270 "renderedHandler"
-app/views/topology/topology.js:289 "setter"
-app/views/topology/topology.js:296 "setter"
-app/views/topology/topology.js:91 "renderOnce"
-app/views/topology/topology.js:278 "getter"
-app/views/topology/topology.js:51 "initializer"
-app/views/topology/topology.js:293 "getter"
-app/views/topology/topology.js:286 "getter"
-app/views/topology/topology.js:170 "serviceForBox"
-app/views/topology/topology.js:274 "getter"
-app/views/topology/topology.js:140 "computeScales"
-app/views/topology/relation.js:172 "initializer"
-app/views/topology/relation.js:321 "drawRelationGroup"
-app/views/topology/relation.js:213 "processRelation"
-app/views/topology/relation.js:877 "addRelationEnd"
-app/views/topology/relation.js:434 "addRelButtonClicked"
-app/views/topology/relation.js:598 "addRelationDragEnd"
-app/views/topology/relation.js:552 "addRelationDragStart"
-app/views/topology/relation.js:942 "_addRelationCallback"
-app/views/topology/relation.js:464 "addRelation"
-app/views/topology/relation.js:536 "snapOutOfService"
-app/views/topology/relation.js:787 "ambiguousAddRelationCheck"
-app/views/topology/relation.js:578 "addRelationDrag"
-app/views/topology/relation.js:209 "renderedHandler"
-app/views/topology/relation.js:392 "drawRelation"
-app/views/topology/relation.js:415 "updateSubordinateRelationsCount"
-app/views/topology/relation.js:693 "cancelRelationBuild"
-app/views/topology/relation.js:982 "subordinateRelationsForService"
-app/views/topology/relation.js:642 "_removeRelationCallback"
-app/views/topology/relation.js:507 "snapToService"
-app/views/topology/relation.js:177 "render"
-app/views/topology/relation.js:427 "draglineClicked"
-app/views/topology/relation.js:255 "updateLinks"
-app/views/topology/relation.js:771 "addRelationStart"
-app/views/topology/relation.js:182 "update"
-app/views/topology/service.js:1384 "fade"
-app/views/topology/service.js:1372 "show"
-app/views/topology/service.js:1020 "dragend"
-app/views/topology/service.js:597 "serviceMouseLeave"
-app/views/topology/service.js:580 "serviceMouseEnter"
-app/views/topology/service.js:444 "initializer"
-app/views/topology/service.js:1534 "show_service"
-app/views/topology/service.js:1378 "hide"
-app/views/topology/service.js:1406 "updateServiceMenuLocation"
-app/views/topology/service.js:640 "serviceStatusMouseOut"
```

Comment:
because of the changes to the methods I re-ran the `undocumented` make target which generated this